CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_klaviyo v1.0.1-a1
+[PR #53](https://github.com/fivetran/dbt_klaviyo/pull/53) includes the following updates:
+
+## Bug Fixes
+- Updated attribution for Shopify order lifecycle events. The following event types now inherit the touch from their corresponding `Placed Order` event, rather than the nearest intervening event.
```

Comment:
Last changelog suggestion: Include the 3-month timeframe guardrail you just added

Reply:
Updated!